votes/initiateNewVote/_EDIT_ME.yml

```
@@ -24,8 +24,9 @@ headerInstructions: |
 #    voters express their preference for each candidates, no matter how many
 #    there are.
 candidates:
-  - TODO
-  - TODO
+  - Status Quo ... Keep distributing corepack with node.js, disabled by default, exactly as it is today
+  - Keep distributing corepack with node.js, enabled by default
+  - Stop distributing corepack with node.js
```

nits
```suggestion
  - Status Quo: keep distributing Corepack with Node.js, disabled by default, exactly as it is today.
  - Keep distributing Corepack with Node.js, enabling `yarn` and `pnpm` by default.
  - Stop distributing Corepack with Node.js.
```
The reason I think it's worth specifying "Yarn and pnpm" is in case e,g, cnpm is added to Corepack, it makes it clear this vote does not cover it.

---

wait I'm confused, why cnpm and other future package managers aren't covered by this vote?

---

What I'm trying to have come across is that, if that's the wining option, collaborators should still be entitled to object to add an additional package manager, so future changes to the Node.js distribution (addition or removal of executables) are still subject to the usual PR review process. Other package managers are certainly somewhat affected by this vote, but this TSC can only make a decision for Corepack "as it is today".

---

I see! yeah, that makes sense 👍 thanks for clarifying!